deploy/charts/rook-ceph/templates/configmap.yaml

```diff
@@ -82,6 +82,12 @@ data:
   ROOK_CSIADDONS_IMAGE: {{ .Values.csi.csiAddons.image | quote }}
 {{- end }}
 {{- end }}
+{{- if .Values.csi.topology }}
+  CSI_ENABLE_TOPOLOGY: {{ .Values.csi.topology.enabled | quote }}
```

Comment:
for now keep this change specific to RBD not a common for both cephfs and rbd

Reply:
all this now only applies to RBD